docs/storage-drivers/gcs.md

```diff
@@ -44,6 +44,17 @@ An implementation of the `storagedriver.StorageDriver` interface which uses Goog
       A private service account key file in JSON format. Instead of a key file <a href="https://developers.google.com/identity/protocols/application-default-credentials">Google Application Default Credentials</a> can be used.
     </td>
   </tr>
+  <tr>
+    <td>
+      <code>credentials</code>
+    </td>
+    <td>
+      no
+    </td>
+    <td>
+      The content of the JSON format key file. The keyfile argument has higher priority than the credentials argument if both are specified.
```

Comment:
I agree with @RichardScothern, in case of a misconfiguration we'd better inform the user. You're right that hardly anyone would specify both, and if they do both parameters are likely contain the same information anyway. However, if not, it is likely to cause the user a lot of head scratching (and associate waste of time) before realising why things aren't working.